### PR TITLE
COMMERCE-12161 Fix checkout bundled product with delta price 0

### PR DESCRIPTION
**Pull request by smottal**
https://liferay.atlassian.net/browse/COMMERCE-12161

### CODE REVIEW

**Comment by smottal** [comment]
ci:test:relevant

**Comment by smottal** [comment]
ci:test:sf

**Comment by smottal** [comment]
ci:test:relevant

**Review by andrea-ale-sbarra — APPROVED** [review]
LGTM.

Thank you!

File: modules/apps/commerce/commerce-service/service.xml [hunk 1]
```diff
@@ -378,6 +378,7 @@
 		<column name="subscriptionType" type="String" />
 		<column name="subscriptionTypeSettings" type="String" />
 		<column name="unitOfMeasureKey" type="String" />
+		<column db-name="unitOfMeasureQuantity" name="unitOfMeasureIncrementalOrderQuantity" type="BigDecimal" />
```
Comment (luca-pellizzon) [inline modules/apps/commerce/commerce-service/service.xml:381]: can we call it `UOMIncrementalOrderQuantity`?

Comment (smottal) [reply modules/apps/commerce/commerce-service/service.xml]: Done

**Comment by smottal** [comment]
ci:test:sf

**Review by luca-pellizzon — APPROVED** [review]

**Comment by lorenzolapescara** [comment]
ci:test:commerce-functional

**Comment by smottal** [comment]
ci:test:relevant

**Comment by smottal** [comment]
ci:test:sf

**Comment by smottal** [comment]
ci:test:commerce

**Comment by smottal** [comment]
ci:test:relevant

**Comment by lorenzolapescara** [comment]
Failures are not related to the PR. @andrea-ale-sbarra can you please force-forward this PR? Thanks!

**Comment by andrea-ale-sbarra** [comment]
ci:forward:force

**Pull request closed without merging**